CoreLib/tests.cpp

```diff
@@ -439,3 +439,68 @@ TEST(CoreLibTests, UniquePtr_CompareWithNullPtr)
     EXPECT_TRUE(nPtr == nPtr);
     EXPECT_FALSE(nPtr != nPtr);
 }
+
+class EmptyDeleter
```

Comment:
Why do we need an empty deleter here in tests?

Reply:
Cuz you should not really delete stack arrays like `int array[10] = {10, 20, 30, 40, 50};`